Submitted By: Tushar Teredesai <tushar@linuxfromscratch.org>
Date: 2003-10-02
Initial Package Version: 1.1.16
Origin: None.
Description: Display compilation messages instead of being silent.
diff -ur cups-1.1.16.orig/Makedefs.in cups-1.1.16/Makedefs.in
--- cups-1.1.16.orig/Makedefs.in	2002-08-22 21:27:40.000000000 -0500
+++ cups-1.1.16/Makedefs.in	2002-10-08 12:13:22.000000000 -0500
@@ -159,7 +159,6 @@
 # Rules...
 #
 
-.SILENT:
 .SUFFIXES:	.a .c .cxx .h .man .o .0 .1 .1m .3 .5 .8 .z
 .c.o:
 	echo Compiling $<...
